EXTENSIONS.md

```diff
@@ -31,7 +31,7 @@ import * as extensionApi from '@podman-desktop/api';
 export async function activate(extensionContext: extensionApi.ExtensionContext): Promise<void> {
   // Create a provider with an example name, ID and icon
   const provider = extensionApi.provider.createProvider({
-    name: 'foobar',
+    name: 'FooBar',
```

Comment:
not sure about this change ?

Reply:
I would like to differentiate the Id and the Name, to make clearer which value is used to reference the provider (the id) and which one is a displayed value only (the name).

Reply:
works for me, I've approved the PR